jetty-ee9/jetty-ee9-security/src/main/java/org/eclipse/jetty/ee9/security/PropertyUserStore.java

```diff
@@ -126,21 +94,37 @@ public Resource getConfigResource()
      */
     public boolean isHotReload()
     {
-        return _hotReload;
+        return getReloadScanSeconds() > 0;
     }
 
     /**
      * Enable Hot Reload of the Property File
      *
-     * @param enable true to enable, false to disable
+     * @param enable true to enable to a 1 second scan, false to disable
+     * @see #setReloadScanSeconds(int)
      */
     public void setHotReload(boolean enable)
```

Comment:
I think this method should go away and be explicitly replaced with `setReloadScanSeconds(1)` calls instead.

Reply:
Need to keep for backwards compatibility.  But I have deprecated. Also discovered that we used to have this attribute and it was called `refreshInterval` so I have renamed to that.